openapi/weaviate/Package.md

```diff
@@ -0,0 +1,18 @@
+Connects to the [Weaviate Vector Search Engine API](https://weaviate.io/developers/weaviate/api) from Ballerina.
+
+## Package overview
+The `ballerinax/weaviate` package is a [Ballerina](https://ballerina.io/) connector for Weaviate Vector Search Engine API. The connector provide access to the manipulations of weaviate schema, objects and search vectors based on various criterias. 
+
+### Compatibility
+|                            | Version                       |
+|----------------------------|-------------------------------|
+| Ballerina Language         | 2201.4.1 (Swan Lake Update 4) |
```

Comment:
Suggested by Jo because Ballerina Central can automatically provide the compatible versions. Even when new releases are available. We can get other's opinion on this too.

Reply:
Let's wait until we get the feedback from Ballerina team since I believe there should be consistency among all connectors we have.

Reply:
Looping in @abeykoon. Could you please share the recommended `module.md`, `pacakge.md` and `README.md` templates for ballerinax packages?